package.json

```diff
@@ -35,100 +35,95 @@
     "cross-env": "^7.0.3",
-    "downshift": "^7.6.0",
     "express": "^4.18.2",
     "fs-extra": "^11.1.1",
     "get-port": "^6.1.2",
     "helmet": "^7.0.0",
     "isbot": "^3.6.10",
     "litefs-js": "^1.1.2",
-    "lru-cache": "^7.18.3",
+    "lru-cache": "^9.1.1",
     "morgan": "^1.10.0",
-    "prisma": "^4.14.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "remix-auth": "^3.4.0",
     "remix-auth-form": "^1.3.0",
     "remix-auth-socials": "^2.0.5",
-    "spin-delay": "^1.2.0",
     "tiny-invariant": "^1.3.1",
     "zod": "^3.21.4"
   },
   "devDependencies": {
-    "@faker-js/faker": "^7.6.0",
-    "@playwright/test": "^1.33.0",
+    "@faker-js/faker": "^8.0.2",
+    "@playwright/test": "^1.34.3",
     "@remix-run/dev": "1.16.1",
     "@remix-run/eslint-config": "1.16.1",
-    "@remix-run/testing": "1.16.1",
-    "@testing-library/dom": "^9.2.0",
+    "@testing-library/dom": "^9.3.0",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^14.0.0",
-    "@testing-library/user-event": "^14.4.3",
     "@total-typescript/ts-reset": "^0.4.2",
     "@types/bcryptjs": "^2.4.2",
     "@types/better-sqlite3": "^7.6.4",
     "@types/compression": "^1.7.2",
-    "@types/cookie": "^0.5.0",
-    "@types/eslint": "^8.37.0",
+    "@types/cookie": "^0.5.1",
+    "@types/eslint": "^8.40.0",
     "@types/express": "^4.17.17",
     "@types/fs-extra": "^11.0.1",
-    "@types/glob": "7.2.0",
+    "@types/glob": "^8.1.0",
     "@types/morgan": "^1.9.4",
-    "@types/node": "^18.16.3",
-    "@types/react": "^18.2.6",
+    "@types/node": "^20.2.5",
+    "@types/react": "^18.2.7",
     "@types/react-dom": "^18.2.4",
-    "@types/testing-library__jest-dom": "^5.14.5",
-    "@types/uuid": "^9.0.1",
+    "@types/testing-library__jest-dom": "^5.14.6",
     "@vitejs/plugin-react": "^4.0.0",
-    "@vitest/coverage-c8": "^0.31.0",
+    "@vitest/coverage-c8": "^0.31.4",
     "autoprefixer": "^10.4.14",
-    "c8": "^7.13.0",
-    "chalk": "^4.1.2",
-    "dotenv": "^16.0.3",
-    "eslint": "^8.40.0",
+    "c8": "^7.14.0",
+    "chalk": "^5.2.0",
+    "dotenv": "^16.1.3",
+    "esbuild": "^0.17.19",
+    "eslint": "^8.41.0",
     "eslint-config-prettier": "^8.8.0",
     "execa": "^7.1.1",
-    "glob": "^7.2.3",
-    "jsdom": "^22.0.0",
+    "glob": "^10.2.6",
+    "jsdom": "^22.1.0",
     "msw": "^1.2.1",
     "npm-run-all": "^4.1.5",
-    "postcss": "^8.4.23",
+    "postcss": "^8.4.24",
     "prettier": "^2.8.8",
     "prettier-plugin-sql": "^0.14.0",
-    "prettier-plugin-tailwindcss": "^0.2.8",
+    "prettier-plugin-tailwindcss": "^0.3.0",
+    "prisma": "^4.15.0",
     "remix-flat-routes": "^0.5.8",
     "tailwindcss": "^3.3.2",
     "tailwindcss-radix": "^2.8.0",
     "tsconfig-paths": "^4.2.0",
     "tsx": "^3.12.7",
     "typescript": "^5.0.4",
-    "vite": "^4.3.5",
+    "vite": "^4.3.9",
     "vite-tsconfig-paths": "^4.2.0",
-    "vitest": "^0.31.0",
+    "vitest": "^0.31.4",
     "vitest-mock-extended": "^1.1.3"
```

Comment:
@kentcdodds,

this is `npx depcheck` output, maybe we can remove some ?
```
Unused dependencies
* remix-auth-socials
Unused devDependencies
* @testing-library/dom
* @testing-library/react
* @types/eslint
* @vitejs/plugin-react
* @vitest/coverage-c8
* autoprefixer
* c8
* postcss
* prettier-plugin-sql
* prettier-plugin-tailwindcss
* tailwindcss
* tailwindcss-radix
* tsconfig-paths
* vitest-mock-extended
```

Reply:
This is tough. Some of these we don't need today and I'm fine removing them. Others are used, but depcheck can't tell (like prettier/tailwind/c8/vitest coverage stuff). While others aren't used in the project but people will need when they start building out features (like testing library stuff).

I think the only thing we can safely remove from that list is remix-auth-socials.